scripts/validate_documentation.py

```diff
@@ -0,0 +1,227 @@
+import argparse
```

Comment:
We should have a comment at the beginning of the script explaining what it does and how to use it.